src/app/pagination/examples/pagination-example.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="padding-15">
```

Comment:
A couple people have asked for a pagination example that demos a list component. I'd like to rename this file as pagination-basic-example.component.html and add a tab, so we can show a simple list example (e.g., pagination-list-example.component.html).

If you're not up for creating that, it can wait for another PR?

Reply:
I'm working on this. I have a work in progress branch for this 
https://github.com/sahil143/patternfly-ng/tree/pagination-list/src/app/pagination/examples